cloud_formation/configs/api.py

```diff
@@ -147,11 +148,13 @@ def create_config(bosslet_config, db_config={}):
     # Queue that holds S3 object keys of cuboids to be indexed.
     config.add_sqs_queue(names.index_cuboids_keys.sqs, names.index_cuboids_keys.sqs, 120, 20160)
 
-    #config.add_sqs_queue("DeadLetterQueue", names.deadletter.sqs, 30, 20160) DP XXX
     config.add_sqs_queue(names.deadletter.sqs, names.deadletter.sqs, 30, 20160)
 
+    # ToDo: determine if a dlq needed for this queue.
+    # Downsample jobs.
+    config.add_sqs_queue(names.downsample_queue.sqs, names.downsample_queue.sqs, 300, 20160)
```

Comment:
What does the 300 second hide timeout do in the case of the downsample queue? Does it mean that if a downsample fails you cant start again for 300 seconds?

Reply:
The message is hidden/unavailable for this amount of time. This prevents another worker from also starting work on the same job. This timeout should be set to longer than it takes to finish the job. If the timeout occurs, this should indicate that the worker crashed, so the message should be made available for other workers.